spec/features/admin/poll/questions_spec.rb

```diff
@@ -111,22 +111,4 @@
 
   pending "Mark all city by default when creating a poll question from a successful proposal"
 
-  it_behaves_like "nested documentable",
-                  "administrator",
-                  "poll_question",
-                  "new_admin_question_path",
-                  { },
-                  "documentable_fill_new_valid_poll_question",
```

Comment:
Hi! I think the methods that after this deletion are not used anywhere else should be removed:
https://github.com/consul/consul/blob/master/spec/shared/features/nested_documentable.rb#L322